mindmeld/constants.py

```diff
@@ -321,9 +321,7 @@
 CURRENCY_SYMBOLS = u"".join(
     chr(i) for i in range(0xFFFF) if unicodedata.category(chr(i)) == "Sc"
 )
-ASCII_CUTOFF = ord("\u0080")
```

Comment:
why do we remove this?

Reply:
Moved it to line 26 of `text_preparation_pipeline/normalizers.py` since it is solely being used by this file. Should I move it to `constants.py` instead?